packages/cli/src/commands/sharedOptions.ts

```diff
@@ -0,0 +1,42 @@
+import { Dictionary } from '@stoplight/types';
+import { Options } from 'yargs';
+
+const sharedOptions: Dictionary<Options> = {
+  port: {
+    alias: 'p',
+    description: 'Port that Prism will run on.',
+    default: 4010,
+    demandOption: true,
+    number: true,
+  },
+
+  host: {
+    alias: 'h',
+    description: 'Host that Prism will listen to.',
+    default: '127.0.0.1',
+    demandOption: true,
+    string: true,
+  },
+
+  cors: {
+    description: 'Enables CORS headers.',
+    boolean: true,
+    default: true,
+  },
+
+  multiprocess: {
+    alias: 'm',
+    description: 'Forks the http server from the CLI for faster log processing.',
+    boolean: true,
+    default: process.env.NODE_ENV === 'production',
+  },
+
+  errors: {
+    description: 'Specifies whether response violations will be sent as an error response',
```

Comment:
```suggestion
    description: 'if among all the violations there's one that's an error, then return an error response instead of the original one',
```